packages/lab/src/calendar/internal/CalendarDay.css

```diff
@@ -1,18 +1,10 @@
-.uitk-light .uitkCalendarDay {
-  --calendar-day-current-day-border-color: var(--uitk-color-grey-900);
-}
-
-.uitk-dark .uitkCalendarDay {
-  --calendar-day-current-day-border-color: var(--uitk-color-white);
-}
-
 /* TODO: Design need to align characteristics for CalendarDay */
 .uitkCalendarDay {
   --calendar-day-text-color: var(--uitk-text-primary-foreground);
   --calendar-day-background: var(--uitk-selectable-background);
   --calendar-day-background-hover: var(--uitk-selectable-background-hover);
   --calendar-day-text-color-hover: var(--uitk-selectable-foreground-hover);
-  --calendar-day-out-of-range-text-color: var(--uitk-text-secondary-foreground);
+  --calendar-day-out-of-range-text-color: var(--uitk-text-secondary-color-disabled);
```

Comment:
might be best to make this token ..-outOfRange-.. as it's the 'type' part of the token (like 'blocked')